src/components/editor/EditorArticleController.jsx

```diff
@@ -13,109 +14,143 @@ export default class EditorArticleController extends FalcorController {
   constructor(props) {
     super(props);
     this.handleSaveChanges = this.handleSaveChanges.bind(this);
-    this.mainFormChanged = this.mainFormChanged.bind(this);
-    this.handleMainFormChange = this.handleMainFormChange.bind(this);
-    this.authorListChanged = this.authorListChanged.bind(this);
+    this.handleFormChanges = this.handleFormChanges.bind(this);
     this.safeSetState({
       changed: "none",
       saving: false
     });
+    this.debouncedHandleFormChanges = debounce(this.handleFormChanges, 500);
```

Comment:
Yeah I had considered it but thought it might be more readable like this. Your point is valid though. I'll change it.